### ISSUE TITLE
will your code work for categorical data?

### ISSUE THREAD

**Issue by Sandy4321** [MLWave/Online-Random-Bit-Regression-FTRL#2]
will your code work for categorical data?

**Comment by MLWave** [comment]
Yeah, if you encode it sanely. Onehot-encoding or target-encoding should work. But care must be taking with target-encoding not to overfit.
